### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @Furry4

### CODE REVIEW

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            
+            Console.Write("Введите числовое значение A : ");
+            string Value_A_text = Console.ReadLine();
+            Value_A_text = Value_A_text.Replace(".", ",");//смена точки на запятую при вводе с клавиатуры пользователем
+            double Value_A = double.Parse(Value_A_text);
+
+            double check = Math.Pow(Value_A, 15);
+            double borders_positive = Math.Pow(1.7, 308);
+            double borders_negative = Math.Pow(1.7, -308);
```
Comment (OverSx) [inline ConsoleApp1/ConsoleApp1/Program.cs:26]: При проверке, достаточно применить метод Math.Abs() для значения переменной check и тогда не нужно вводить новую переменную для проверки отрицательного значения.

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+            if (check > borders_positive || check < borders_negative)
+            {
+                Console.WriteLine("Error!\nThe user entered too large (small) number");
+            }
+
+            else {
+                double Value_B = Math.Round(Value_A * Value_A,3);//где Value_B значение Value_A в степени 2
```
Comment (OverSx) [inline ConsoleApp1/ConsoleApp1/Program.cs:34]: Для умножения переменной саму на себя сколько угодно раз удобнее использовать метод Math.Pow(value, degree)

**Comment by OverSx** [comment]
При вычислении a^2, a^3 и тд. лучше использовать Math.Round() непосредственно при выводе значения в консоль, так дальнейшие вычисления будут точнее.

**Comment by OverSx** [comment]
Оценка: 4/4

**Comment by Furry4** [comment]
Thx, в следующий раз постараюсь использовать эти функции

**Pull request merged**